github.go

```diff
@@ -29,15 +31,16 @@ type githubImporter struct {
 // @Produce json
 // @Param import body importBody _ "import object"
 // @Success 201 "Successfully imported"
-// @Failure 400 {object} string "Bad request"
-// @Failure 401 {object} string "Unauthorized or missing jwt token"
+// @Failure 400 {object} problem.Response "Bad request"
+// @Failure 401 {object} problem.Response "Unauthorized or missing jwt token"
+// @Failure 502 {object} problem.Response "Bad gateway"
 // @Router /github [post]
 func runGitHubHandler(c *gin.Context) {
 	i := importBody{}
 	err := c.BindJSON(&i)
 	if err != nil {
-		c.Error(err)
-		c.JSON(http.StatusBadRequest, err.Error())
+		ginutil.WriteInvalidBindError(c, err,
```

Comment:
Need to use `c.ShouldBindJSON` now that you're using problem responses

https://iver-wharf.github.io/#/development/gin-shouldbind-vs-bind